users.rb

```diff
@@ -0,0 +1,18 @@
+require './database_setup.rb'
+
+class User
+  attr_accessor :name,:email,:password,:user_type
+
+  def initialize(name,email,password,user_type)
+    self.name=name
+    self.email=email
+    self.password=password
+    self.user_type=user_type
+  end
+
+
+  def save 
+    connection=PG.connect(dbname: 'postgres',user: 'lakshayjain' , host: 'localhost')
```

Comment:
can we make a small class that takes care of DB related things so that connecting to DB is easy for other classes?

Reply:
we are creating a new connection everytime we want to do something, you can think of singleton object object here for db connection